### PR TITLE
Create issue template - #797

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Your issue may already be reported!
+Please search on the [issue track](https://github.com/arturbosch/detekt/issues) before creating one.
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
```
Comment (schalkms) [inline ISSUE_TEMPLATE.md]: The current

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<!--- If suggesting a change/improvement, explain the difference from the current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
```
Comment (Mauin) [inline ISSUE_TEMPLATE.md]: Is the list really needed? I'd assume bug reports should mostly just have a block of code.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from the current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
```
Comment (Mauin) [inline ISSUE_TEMPLATE.md]: Not super sure if this section is required. Reporters can still add comments/text below the template

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+Your issue may already be reported!
+Please search on the [issue track](https://github.com/arturbosch/detekt/issues) before creating one.
```
Comment (Mauin) [inline ISSUE_TEMPLATE.md]: I'd replace "on the issue track" with "please search in the issues".

**Comment by arturbosch** [comment]
KK, addressed your feedback. I will squash the commits when rebasing. Oh and I myself used a template for this template, thanks for extra checking :)

**Pull request merged**